docs/src/index.md

```diff
@@ -1,4 +1,4 @@
-# StatsBase.jl Documentation
+# Getting Started
```

Comment:
"Getting Started" doesn't sound appropriate given that this page essentially gives the table of contents. I'd keep the existing name.